dali/test/python/test_operator_file_reader.py

```diff
@@ -0,0 +1,116 @@
+import tempfile
+import os
+import nvidia.dali.fn as fn
+from functools import partial
+from nvidia.dali.pipeline import Pipeline
+
+def ref_contents(path):
+    fname = path[path.rfind('/')+1:]
+    return "Contents of " + fname + ".\n"
+
+def populate(root, files):
+    for fname in files:
+        with open(os.path.join(root, fname), "w") as f:
+            f.write(ref_contents(fname))
+
+g_root = None
+g_tmpdir = None
+g_files = None
+
+def setup_module():
+    global g_root
+    global g_files
+    global g_tmpdir
+
+    g_tmpdir = tempfile.TemporaryDirectory()
+    g_root = g_tmpdir.__enter__()
+    g_files = [str(i)+'.dat' for i in range(10)]
+    populate(g_root, g_files)
+
+def teardown_module():
+    global g_root
+    global g_files
+    global g_tmpdir
+
+    for f in g_files:
```

Comment:
Do you need that? Won't `g_tmpdir.__exit__(None, None, None)` clean the dir?

Reply:
I don't know....